samples/simple/index.html

```diff
@@ -0,0 +1,9 @@
+<!doctype html>
```

Comment:
Do we need this `index.html` and the one in `webRoot`?

Reply:
Well I like to not edit anything in webRoot directly and treat it all as generated.. You'll notice that the new build copies the index.html from the source root (which is still the project root) every time you compile.. so changes should be made in sources not in webRoot (output).